flow2ml/Data_Augumentation.py

```diff
@@ -109,4 +109,18 @@ def applyShear(self,classPath):
             # saving the image by
             plt.imsave(classPath+"/ShearedImages/Sheared"+image, cv2.cvtColor(Sheared, cv2.COLOR_RGB2BGR))
           except Exception as e:
-            print(f"Shearing operation failed due to {e}")
\ No newline at end of file
+            print(f"Shearing operation failed due to {e}")
+
+  
+  def HistogramEqualisation(image_file):
+    '''class containing method to implement histogram equalization to a given image.'''
+    #read the image and flag is 0 here,to indicate image is loaded in grayscale mode.
+    img=cv2.imread(image_file,0) 
+    #Equalize the Histogram for contrast adjustment.
+    equalised_img=cv2.equalizeHist(img)
+    #stacking both the original and equalised images.
+    result=np.hstack((img,equalised_img))
+    #final image is in output.png file
+    cv2.imwrite("output.png",result)
```

Comment:
Follow the directory structure that other augmentation methods are using to save processedImages.